docs/docs/prerender.md

```diff
@@ -49,23 +49,51 @@ You can also prerender your not found page (a.k.a your 404 page). Just add—you
 
 This will prerender your NotFoundPage to `404.html` in your dist folder. Note that there's no need to specify a path.
 
-## Cells, Private Routes, and Dynamic URLs
+## Private Routes
 
-How does Prerendering handle dynamic data? For Cells, Redwood prerenders your Cells' `<Loading/>` component. Similarly, for Private Routes, Redwood prerenders your Private Routes' `whileLoadingAuth` prop:
+For Private Routes, Redwood prerenders your Private Routes' `whileLoadingAuth` prop:
 
-```jsx {1,2}
+```jsx
 <Private >
   // Loading is shown while we're checking to see if the user's logged in
   <Route path="/super-secret-admin-dashboard" page={SuperSecretAdminDashboard} name="ssad" whileLoadingAuth={() => <Loading />} prerender/>
 </Private>
 ```
 
-Right now prerendering won't work for dynamic URLs. We're working on this. If you try to prerender one of them, nothing will break, but nothing happens.
+## Dynamic routes
 
-```jsx title="web/src/Routes.js"
+Let's say you have a route like this
+
+```jsx
 <Route path="/blog-post/{id}" page={BlogPostPage} name="blogPost" prerender />
 ```
 
+To be able to prerender this route you need to let Redwood know what `id`s to use. You do this by creating a `BlogPostPage.renderData.js` file next to the page file itself (so next to `BlogPostPage.js` in this case). It should export a function called `routeParameters` that returns an array of objects with the path param name as the key, and the value for the parameter as the object value. A single Page component can be used for different routes. The name of the current route will be passed as an argument to `routeParameters` so you can return different route parameters depending on what route it is, if you need to. An example will hopefully make all this clearer.
+
+For the example route above, all you need is this:
+
+```js title="BlogPostPage.renderData.js"
+export function routeParameters() {
+  return [{ id: 1 }, { id: 2 }, { id: 3 }]
+}
+```
+
+Of, if you wanted to get fancy
+
+```js title="BlogPostPage.renderData.js"
+export function routeParameters(routeName) {
```

Comment:
```suggestion
export function routeParameters({ routeName }) {
```

Reply:
The destructured parameter name would actually just be `name` with the way I implemented it. So to keep it clear that it's the route's name (and not some other random name), I decided to not destructure and just refer to it as `route.name` when accessing it instead